nuxeo-web-ui/Jenkinsfiles/server-web-ui.groovy

```diff
@@ -0,0 +1,318 @@
+/*
+* (C) Copyright 2022 Nuxeo (http://nuxeo.com/) and others.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* Contributors:
+*     Thomas Fowley
+*/
+CONNECT_PROD_URL = 'https://connect.nuxeo.com/nuxeo/site/'
+CONNECT_PREPROD_URL = 'https://nos-preprod-connect.nuxeocloud.com/nuxeo/site/'
```

Comment:
These variables should be available in the environment, see https://github.com/nuxeo/platform-ci/blob/master/charts/jenkins/values-casc-jenkins.yaml.gotmpl#L19-L22